CloudAppSecurityDocs/connect-zendesk-to-microsoft-cloud-app-security.md

```diff
@@ -46,7 +46,7 @@ This article provides instructions for connecting Microsoft Cloud App Security t
 ### Configure Cloud App Security
 
 >[!NOTE]
->The Zendesk admin needs to be logged into their Zendesk organization within their browser before installing the connector from the Cloud App Security portal.
+>The Zendesk user that is configuring the integration needs to be alwayes Zendesk admin.
```

Comment:
What do you mean "always needs to be a Zendesk admin"? Why can't we just say "needs to be a Zendesk admin"?

Reply:
We have seen customers that downgrade permissions of the Zendesk user after the connection. Therefore, we want to emphasize that the Zendesk user should remain an admin.

Reply:
So how about, "The Zendesk user that is configuring the integration must always remain a Zendesk admin, even after the connector is installed."?

Reply:
@dcurwin  Sounds much better :)